bin/vm.py

```diff
@@ -91,16 +111,7 @@ def qemu_args(
 
         # Command-line arguments.
         if build_path is not None:
-            newconfig = subprocess.check_output(
-                ["make", "-s", "listnewconfig"], cwd=build_path, universal_newlines=True
-            ).strip()
-            if newconfig:
-                sys.exit(
-                    "Kernel build .config is not up to date; cannot determine image name"
-                )
```

Comment:
torvalds/linux@993bdde94547887faaad4a97f0b0480a6da271c3 made this unnecessary, so I think we can just delete the whole newconfig check and be done with it.

Reply:
Sounds good to me, will send a v2 patch